.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,37 @@
+---
+name: Continuous Integration
+
+on:
+  push:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
```

Should we be testing against all of the ruby versions our services run on?
```suggestion
    strategy:
      fail-fast: false
      matrix:
        ruby: [2.6, 2.7, '3.0', 3.1]
    runs-on: ubuntu-18.04
    steps:
      - uses: actions/checkout@v2
      - name: Set up Ruby
        uses: ruby/setup-ruby@v1
        with:
          ruby-version: ${{ matrix.ruby }}
          bundler-cache: true
```